src/components/TeamGeneratorPage/TeamsDisplay/ImportPlayerList.tsx

```diff
@@ -1,22 +1,26 @@
 import React, { useState, useEffect, useContext } from 'react';
-import { DisplayContext } from './services/Context';
+import { DisplayContext } from '../services/Context';
 
-const ImportPlayerList = ({ onImportPlayerList }) => {
-	const [playerList, setPlayerList] = useState('');
+interface ImportPlayerListProps {
```

Comment:
this can be a generic interface because you use it in the AddRemoveTeam component